static-page/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+
+<html lang="en" class="govuk-template  flexbox no-flexboxtweener">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
+    <title>GOV.UK Pay API v1.0.3</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+    <meta name="theme-color" content="#0b0c0c">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="robots" content="noindex, nofollow">
+    <link href="./resources/main.css" rel="stylesheet" media="all">
+</head>
+
+<body class="govuk-template__body">
+
+<header class="govuk-header " role="banner" data-module="govuk-header">
+    <div class="govuk-header__container govuk-width-container">
+        <div class="govuk-header__logo">
+            <a href="https://govukpay-api-browser.cloudapps.digital/"
```

Comment:
should the logo link to the product pages instead?

Reply:
hmm not sure. We have mixed use case (selfservice - to same app, tech docs - to product pages)

Reply:
My only thought is because there's nothing on this page, maybe it should link elsewhere. Although it doesn't seem very important either way.